internal/database/metadata/mysql/entity.go

```diff
@@ -14,15 +14,15 @@ func createEntity(ctx context.Context, sqlxCtx metadata.SqlxContext, opt metadat
 	if err != nil {
 		if er, ok := err.(*mysql.MySQLError); ok {
 			if er.Number == ER_DUP_ENTRY {
-				return 0, fmt.Errorf("entity %s already exists", opt.EntityName)
+				return 0, errors.Errorf("entity %s already exists", opt.EntityName)
 			}
 		}
 		return 0, err
```

Comment:
missing wrap ?

Reply:
`errors.Errorf` will automatically save the call stack, just like `errors.WithStack`.

Reply:
fixed